### ISSUE TITLE
channel tag sorting index is being ignored

### ISSUE THREAD

**Issue by horstepipe** [zipleen/tvheadend-iphone-client#232]
Hey
The tvos app shows the categories in alphabetical order regardless of a sorting index is being set serverside.

Best regards

**Comment by zipleen** [comment]
sorting set serverside ? what do you mean? 

tvOS app has a sorting option in settings - navigate to setting and choose sorting by name or number!

**Comment by horstepipe** [comment]
yeah that sorts the channels itself, I'm talking about the categories/tags. You can give them a sorting index number serverside, which is being ignored by the client

**Comment by horstepipe** [comment]
I'm talking about the tvos version if this is of importance

**Comment by zipleen** [comment]
Oh!

Didn't notice! I shall look into this soon then

**Comment by horstepipe** [comment]
if the field is not filled you could fallback to the default alphabetical order as it works currently
